app/views/admin/tags/index.html.erb

```diff
@@ -15,48 +15,55 @@
-            <% LocaleLanguage.all.each do |language| %>
-              <div id=<%="#{language.iso_code}_#{tag.id}"%> >
-                <%= check_box_tag 'languages[]', language.iso_code,
-                  tag.locale_languages.map(&:iso_code).include?("#{language.iso_code}"),
-                  id: tag.name + "_" + language.iso_code %>
-                <%= label_tag t(language.iso_code, scope: 'i18n.language') %>
-              </div>
+        <th><%= sortable "tags.name", "Name" %></th>
+        <th><%= t(:categories, scope: 'admin.dashboard.tags') %></th>
+        <th><%= t(:languages, scope: 'admin.dashboard.tags') %></th>
+        <th>Profiles</th>
+        <th><%= sortable 'taggings.created_at', t(:created_at, scope: 'admin.dashboard.tags') %></th>
+        <th><%= t(:delete, scope: 'admin.dashboard.tags') %></th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @tags.each do |tag| %>
+        <tr>
+          <td>
+            <b><%= link_to tag.name, edit_admin_tag_path(tag.id, page: params[:page]),
+            data: { toggle: 'tooltip', placement: 'left' }, title: 'Edit' %></b></b>
+            <%= link_to '<i class="fa fa-pencil">&nbsp;</i>'.html_safe, edit_admin_tag_path(tag.id, page: params[:page]),
+            data: { toggle: 'tooltip', placement: 'left' }, title: 'Edit' %>
+          </td>
+          <td>
+            <% tag.categories.pluck(:name).each do |name| %>
+              <%= name %> <br/>
+            <% end %>
+          </td>
+          <td>
+            <% tag.locale_languages.pluck(:iso_code).each do |lang| %>
+              <%= t(lang, scope: 'i18n.language') %><br/>
             <% end %>
-            <%= f.button :submit, t(:save, scope: 'general'), class:'btn btn-blue btn-sm' %>
-          <% end %>
-        </td>
-        <% @categories.each do |category| %>
+          </td>
           <td>
-            <% if tag.categories.pluck(:id).include?(category.id) %>
-              <%= link_to category.name, remove_category_admin_tag_path(tag,
-              category_id: category.id, page: params[:page], q: params[:q],
-              uncategorized: params[:uncategorized]), method: :post, class: 'color-white bg-primary px-1'  %>
-            <% else %>
-              <%= link_to category.name, set_category_admin_tag_path(tag,
-              category_id: category.id, page: params[:page], q: params[:q],
-              uncategorized: params[:uncategorized]), method: :post %>
+            <% tag.taggings.each do |tagging| %>
+              <%= link_to tagging.taggable.name_or_email, admin_profile_path(tagging.taggable),
+                  html_options = { title: t(:show, scope: 'admin.dashboard.profiles.edit') } %><br/>
+            <% end %>
+          </td>
+          <td>
+            <% if  !tag.taggings.empty? %>
+              <%= tag.taggings.sort_by{|t| t.created_at }.last.created_at.strftime('%d.%m.%Y') %>
```

Comment:
`ActsAsTaggableOn::Tag.first.taggings.first.created_at` I would change the last tagging to first tagging to get the creation date of the tag.

Reply:
Changed tagging to be the first one. I think for this pull request everything is ready

Reply:
Thank you!!!